### PR TITLE
Add BigQuery reference doc.

### PR DESCRIPTION
**Pull request by pauloouriques**

### CODE REVIEW

File: data/plugins/googleBigQuery.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Plugin } from "./index";
 
 export const GoogleBigQuery: Plugin = {
-  name: "Google BigQuery",
```
Comment (evantahler) [inline data/plugins/googleBigQuery.ts:4]: @andyjih this would change the name of the plugin on the other pages as well.  What do you think?

**Review by evantahler — COMMENTED** [review]
👍 from me, but let's wait for @andyjih on the name change

File: data/plugins/googleBigQuery.ts (hunk 1 above)
Comment (pauloouriques) [reply data/plugins/googleBigQuery.ts:4]: I thought the same thing, but I looked around and all looks fine to me.

**Comment by evantahler** [comment]
@andyjih I'm merging this in with the name change - we can revert if it's a problem

**Pull request merged**